source/LimitedEvents.h

```diff
@@ -0,0 +1,110 @@
+/* LimitedEvents.h
+Copyright (c) 2023 by an anonymous author.
```

Comment:
uh, you sure about this?
with git blame everybody can see your github account anyways. might as well specify it here.

wont force you to do that though

Reply:
It is a jab at @NomadicVolcano (my sister) who rants loudly at me if I put my name in a file. Everyone knows it is me since I'm the only one who writes "an anonymous author," but technically my name isn't in there.